src/index.html

```diff
@@ -1,13 +1,42 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/style.css">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="imac"
+        class="product-card__img"
+      >
+      <div class="product-card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+      <div class="product-card__code">Product code: 195434</div>
+      <div class="product-card__response">
```

Comment:
Add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements
![2023-02-10_15h03_21](https://user-images.githubusercontent.com/61904995/218100414-694acd07-03a1-4cd0-8b46-af85b207741e.png)